pkg/reconciler/v1alpha1/taskrun/resources/pod.go

```diff
@@ -134,14 +140,14 @@ func gcsToContainer(source v1alpha1.SourceSpec, index int) (*corev1.Container, e
 		Env:          implicitEnvVars,
 	}, nil
 }
+*/
```

Comment:
looks like we can probably delete this now?

Reply:
wait… I was sure I removed it :thinking:

Reply:
it is the zombie deleted code O.O

Reply:
undead code